src/config/default.json

```diff
@@ -6,5 +6,5 @@
     },
     "Port": "4000",
     "LOG_LEVEL": "info",
-    "GRAPHS_SYNC_TIMEOUT": "600000" 
+    "GRAPHS_SYNC_INTERVAL": "60000" 
```

Comment:
Mind to explain this change? It seems unrelated.

Reply:
2 changes here.

First is the variable name, instead of using TIMEOUT use INTERVAL, as it is the interval the app is going to use to fetch data from the registry.

Second is the interval length, I was using 10 minutes, but looks like it could be too much, so I changed it to 1 minute